ansible/roles/machine/provision/templates/hosts

```diff
@@ -2,7 +2,12 @@
 ::1  localhost
 {% for host in groups['all'] %}
    {% if 'ansible_default_ipv4' in hostvars[host] %}
-      {{ hostvars[host]['ansible_default_ipv4']['address'] }} {{ hostvars[host]['inventory_hostname'] }}.ipa.test
+      {% if host.startswith('controller') or host.startswith('master') or host.startswith('replica') or host.startswith('client') %}
+            {{ hostvars[host]['ansible_default_ipv4']['address'] }} {{ hostvars[host]['inventory_hostname'] }}.ipa.test
+      {% endif %}
+      {% if host.startswith('trustedmaster') or host.startswith('trustedreplica') or host.startswith('trustedclient') %}
+            {{ hostvars[host]['ansible_default_ipv4']['address'] }} {{ hostvars[host]['inventory_hostname'] }}.trustedipa.test
+      {% endif %}
```

Comment:
All the above code should be aligned left, otherwise the /etc/hosts file will contain indented lines as can be seen [here](http://freeipa-org-pr-ci.s3-website.eu-central-1.amazonaws.com/jobs/38daad96-6abb-11ef-9899-fa163e320d5d/test_integration-test_multidomain_ipa.py-TestMultidomain-test_multidomain_setup/master.ipa.test/etc/hosts.gz)